rustls/src/server/hs.rs

```diff
@@ -476,6 +477,7 @@ impl State for ExpectClientHello {
             }
             .handle_client_hello(suite, conn, certkey, &m)
         } else {
+            let suite = suites::Tls12CipherSuite::try_from(suite).unwrap();
```

Comment:
In the future, the selection logic for `suitable_suites` should be refactored to eliminate this unwrap.